### PR TITLE
Add avatar zip uploading and import from shared

### PR DESCRIPTION
**Pull request by k7hpn**
- Add uploading an avatar packate .zip file
- Add import of avatar .zip file from shared/private/avatars.zip

### CODE REVIEW

File: src/GRA.Controllers/MissionControl/AvatarsController.cs [hunk 1]
```diff
@@ -498,5 +493,126 @@ public async Task<IActionResult> BundleDelete(int id)
...
+            if (Directory.Exists(assetPath))
+            {
+                Directory.Delete(assetPath, true);
+            }
+
+            Directory.CreateDirectory(assetPath);
+
+            try
+            {
+                ZipFile.ExtractToDirectory(avatarZip, assetPath);
```
Comment (iafb) [inline src/GRA.Controllers/MissionControl/AvatarsController.cs:584]: Is giving an error saying "Zip entry name ends in directory separator character but contains data."

File: src/GRA.Controllers/MissionControl/AvatarsController.cs [hunk 2]
```diff
@@ -498,5 +493,126 @@ public async Task<IActionResult> BundleDelete(int id)
 
             return PartialView("_ItemsPartial", viewModel);
         }
+
+        [HttpPost]
+        public async Task<IActionResult> SetupAvatars(AvatarIndexViewModel viewModel)
```
Comment (iafb) [inline src/GRA.Controllers/MissionControl/AvatarsController.cs:498]: Probably needs an increase in request size limit, I'm getting errors trying to upload the zip due to how large it is.

File: src/GRA.Controllers/MissionControl/AvatarsController.cs (hunk 1 above)
Comment (k7hpn) [reply src/GRA.Controllers/MissionControl/AvatarsController.cs:584]: It seems like the .NET Core ZIP capabilities don't work well with archives generated by 7-Zip. If you use the right-click -> Compress option in Windows Explorer then the archives upload okay.

File: src/GRA.Controllers/MissionControl/AvatarsController.cs (hunk 2 above)
Comment (k7hpn) [reply src/GRA.Controllers/MissionControl/AvatarsController.cs:498]: I added the capability of pulling in a file from the `shared/private` directory for the large avatar file, the Web form upload is just for smaller files.

**Review by iafb — APPROVED** [review]

**Pull request merged**